Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -1518,23 +1670,48 @@ function zoomCV(controller, startPosition, movement) {
   var camera = scene.camera;
   var canvas = scene.canvas;
 
-  var windowPosition = zoomCVWindowPos;
-  windowPosition.x = canvas.clientWidth / 2;
-  windowPosition.y = canvas.clientHeight / 2;
+  var cameraUnderground = controller._cameraUnderground;
+
+  var windowPosition;
+
+  if (cameraUnderground) {
```

Comment:
I considered it. I'm not sure what will break for above ground navigation. At the same time this could improve camera navigation on the whole.

This could be a good thing to try post June 1st.